apps/entity-service/src/entities/entities.service.ts

```diff
@@ -53,6 +55,10 @@ const ASSOCIATION_PROCESSORS = {
       Demographic.findAll({
         where: { demographicalType, demographicalId, hidden: false },
         include: [{ association: "entries" }]
+      }),
+    ({ id: demographicalId }, demographicalType) =>
+      Demographic.count({
+        where: { demographicalType: demographicalType.toString(), demographicalId, hidden: false }
```

Comment:
Unless I'm missing something, we only need this count accessor for models that intend to support pagination. It's possible we'll want to support pagination for demographics, but if we do, we'd want to save that for a future ticket.

Reply:
removed.